### PR TITLE
chore: react - export missing types

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: packages/react/src/publication/useNotInterestedToggle.ts [hunk 1]
```diff
@@ -37,7 +35,7 @@ export type NotInterestedOperation = UseDeferredTask<void, never, UseNotInterest
  * @category Publications
  * @group Hooks
  */
-export function useNotInterestedToggle(): NotInterestedOperation {
+export function useNotInterestedToggle(): UseDeferredTask<void, never, UseNotInterestedToggleArgs> {
```
Comment (cesarenaldi) [inline packages/react/src/publication/useNotInterestedToggle.ts:38]: No objection per se, just curious about the rationale for removing this alias.

File: packages/react/src/publication/index.ts [hunk 2]
```diff
@@ -19,33 +19,80 @@ export * from './useWhoReactedToPublication';
...
+  ThreeDMetadataV3,
+  ThreeDMetadataV3Asset,
+  TransactionMetadataV3,
   UnknownOpenActionModuleSettings,
   UnknownReferenceModuleSettings,
+  Video,
+  VideoMetadataV3,
+  WhoActedOnPublicationWhere,
+  WhoReactedPublicationWhere,
 } from '@lens-protocol/api-bindings';
```
Comment (cesarenaldi) [inline packages/react/src/publication/index.ts:96]: Good job!

File: packages/react/src/publication/useNotInterestedToggle.ts (hunk 1 above)
Comment (krzysu) [reply packages/react/src/publication/useNotInterestedToggle.ts:38]: I just found them unnecessary, and not consistent with other hooks, it's a breaking change though

Comment (cesarenaldi) [reply packages/react/src/publication/useNotInterestedToggle.ts:38]: Understood. I wouldn't worry too much about a breaking change. I think it's very unlikely anybody referenced these types in code.

**Pull request merged**